src/Microsoft.DotNet.Watcher.Tools/DotNetWatcher.cs

```diff
@@ -45,6 +46,10 @@ public DotNetWatcher(ILogger logger)
                     var fileSetTask = fileSetWatcher.GetChangedFileAsync(combinedCancellationSource.Token);
                     var processTask = _processRunner.RunAsync(processSpec, combinedCancellationSource.Token);
 
+                    _logger.LogInformation("Running {execName} with the following arguments: {args}",
+                        processSpec.ShortDisplayName(),
+                        ArgumentEscaper.EscapeAndConcatenate(processSpec.Arguments));
```

Comment:
The escaping has to account for backslashes, spaces, and quotes. cref https://blogs.msdn.microsoft.com/twistylittlepassagesallalike/2011/04/23/everyone-quotes-command-line-arguments-the-wrong-way/